File: beecrowd/iniciante/c/1043.c
#include <stdio.h>

int main() {
    float a, b, c, trapezoid_area;

    scanf("%f %f %f", &a, &b, &c);

    if ((a + b <= c) || (a + c <= b) || (b + c <= a)) {
        trapezoid_area = (a + b) * c / 2;
        
        printf("Area = %.1f\n", trapezoid_area);
    } else {
        printf("Perimetro = %.1f\n", a + b + c);
    }
    return 0;
}

File: beecrowd/beginner/python/1043.py
a, b, c = map(float, input().split())

if a + b <= c or a + c <= b or b + c <= a:
    print(f'Area = {(a + b) / 2 * c:.1f}')
else:
    print(f'Perimetro = {a + b + c:.1f}')
